src/NetAppFiles/NetAppFiles/help/Update-AzNetAppFilesPool.md

```diff
@@ -0,0 +1,243 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.NetAppFiles.dll-Help.xml
+Module Name: Az.NetAppFiles
+online version: https://docs.microsoft.com/en-us/powershell/module/az.netappfiles/update-aznetappfilespool
+schema: 2.0.0
+---
+
+# Update-AzNetAppFilesPool
+
+## SYNOPSIS
+Updates an Azure NetApp Files (ANF) pool with the new data set.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Update-AzNetAppFilesPool -ResourceGroupName <String> -Location <String> -AccountName <String> -Name <String>
```

Comment:
The question of location has been raised a few times recently. It is understood that it is an ARM requirement that all resources need location specified in the request even if they must be in the same region as their parent, is that correct? If so is this only for create requests? and then for put (update), delete, get and patch it can be omitted?